server-code/Projects.js

```diff
@@ -2,12 +2,48 @@
 Document: JS code specific for projects
 */
 
+const slideDeckStr = "Slide Deck";
+const bomStr = "BOM";
+
 /**
  * Returns HTML formatted list of all projects.
  * 
  * @return {String} – A constructed HTML table listing all the projects
  */
 function getAllProjects() {
     var data = getSheetInfo('mainSheetID', 'Projects', 'data');
+    transformToHyperLinks(data);
     return buildTableHTML(data, "table-sm");
 }
+
+/**
+ * Creates an HTML hyperlink, given the display text, that requests the given url.
+ * 
+ * @param {String} url – The url to link to
+ * @param {String} displayText – The text that will be displayed
+ * @return {String} – The corresponding HTML hyperlink
+ */
+function getHTMLLink(url, displayText) {
+    return displayText.link(url);
```

Comment:
this is cool, I didn't know it existed. unfortunately, [Mozilla MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/link) seems to indicate that it's a deprecated feature, so we should probably just do it manually with a string template

Reply:
Agreed. Was going to point this out just now as well.

Reply:
Resolved by commit 8ad9b33794c32a6aa1b117af787129ffee61e860

Reply:
Actually, resolved by af6f9d4d4973afd66a5224d174052a511297b541